apps/desktop/src/components/Sidebar.tsx

```
@@ -0,0 +1,70 @@
+  const collapsed = useAppStore((s) => s.sidebarCollapsed);
+  const toggleSidebar = useAppStore((s) => s.toggleSidebar);
+
+  return (
+    <nav className={`${collapsed ? 'w-14' : 'w-56'} bg-slate-900 border-r border-slate-800 flex flex-col transition-[width] duration-200`}>
+      <div className="p-4 border-b border-slate-800 flex items-center justify-between">
+        {!collapsed && <span className="text-sm font-semibold text-slate-200">Kata Cloud Agents</span>}
+        <button
+          onClick={toggleSidebar}
+          className="text-slate-400 hover:text-slate-200"
+          aria-label={collapsed ? 'Expand sidebar' : 'Collapse sidebar'}
+        >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)